docs/drilldowns/explore-underlying-data.asciidoc

```diff
@@ -0,0 +1,45 @@
+[[explore-underlying-data]]
+== Explore data underlying visualizations
+
+Dashboard panels have "Explore underlying data" actions that will redirect
+you to *Discover* narrowing down documents to ones likely used in
+the visualization.
+
+"Explore underlying data" actions are available for visualizations which are
+backed by a single index pattern.
+
+There are two "Explore underlying data" actions: (1) one which appears in the
+panel's context menu; (2) and the other which appears when user clicks or brushes
+the chart.
+
+[float]
+[[explore-data-from-panel-context-menu]]
+=== Explore data from panel context menu
+
+Context menu "Explore underlying data" action will navigate you to *Discover*,
+to the index pattern used in the visualization and will carry over all filters,
+query, and time range applied to the visualization.
+
+[role="screenshot"]
+image::images/explore_data_context_menu.png[Explore underlying data from panel context menu]
+
+[float]
+[[explore-data-from-chart]]
+=== Explore data from chart action
+
+In-chart "Explore underlying data" action will also navigate you to *Discover*,
+to the index pattern used in the visualization and will carry over all
+filters, query and time range, but it will also apply the filters and time range
+created from the click or brush events that triggered this action. For example,
```

Comment:
I think it may be better to reference a specific example here, as illustrated by the image below. I.e. describe what would happen, if the user selected this action after clicking on that specific slice of the chart.

Reply:
++

Reply:
I've replaced this paragraph with the one Gail suggested above. I think it is good now, tell me if you think we should still provide an example.